azurelinuxagent/daemon/scvmm.py

```diff
@@ -70,4 +71,5 @@ def run(self):
         if self.detect_scvmm_env():
             self.start_scvmm_agent()
```

Comment:
@ahmetalpbalkan scvmm does not (currently) have systemd unit. It definitely should.